backend/src/server/routes/v1/project-router.ts

```diff
@@ -511,4 +512,52 @@ export const registerProjectRouter = async (server: FastifyZodProvider) => {
       return { serviceTokenData };
     }
   });
+
+  server.route({
+    method: "GET",
+    url: "/id",
+    config: {
+      rateLimit: readLimit
+    },
+    schema: {
+      querystring: z.object({
+        slug: z.string().trim().optional().describe(PROJECTS.GET_ID.slug),
+        name: z.string().trim().optional().describe(PROJECTS.GET_ID.name)
```

Comment:
Yah this will be a bad idea because name can be duplicate in an organization. Will an org have duplicate project name not sure how often. But a name can be duplicate but slug will never be.